src/screens/ReceiveCoinsScreen.tsx

```diff
@@ -49,7 +50,7 @@ class ReceiveCoinsScreen extends Component<Props, State> {
   }
 
   updateAmount = (amount: string) => {
-    const parsedAmount = amount.replace(',', '.');
+    const parsedAmount = amount;
```

Comment:
`const parsedAmount = parseFloat(parsedAmount);` I guess

Reply:
so I guess is logic from @Kamill90  are u accept ?

Reply:
sure

Reply:
I mean sure we can change it but it seems to work for the last 8 months and I am not sure if this replace was needed for some edge case, would need to test it first.